code-style.md

```diff
@@ -0,0 +1,9 @@
+## Flowing Code Style Guide / 1.0.0
+
+### Java
+
+Apply the [Google Java Style Guide](https://google.github.io/styleguide/javaguide.html)
+
+* Eclipse: download the [eclipse-java-google-style](https://github.com/google/styleguide/blob/gh-pages/eclipse-java-google-style.xml) formatter.
+* IntelliJ: install the google-java-format plugin.
```

Comment:
We can also add vscode:
VSCode: Use google's eclipse formatter as explained [here](https://code.visualstudio.com/docs/java/java-linting#_formatter)

Reply:
Excellent!